Content.Server/Atmos/Piping/Unary/EntitySystems/GasThermoMachineSystem.cs

```diff
@@ -78,25 +80,32 @@ private void OnGasThermoRefreshParts(EntityUid uid, GasThermoMachineComponent co
                 switch (part.PartType)
                 {
                     case MachinePart.MatterBin:
+                        nBins += 1;
                         matterBinRating += part.Rating;
                         break;
                     case MachinePart.Laser:
+                        nLasers += 1;
                         laserRating += part.Rating;
                         break;
                 }
             }
+            laserRating /= nLasers;
+            matterBinRating /= nBins;
 
-            component.HeatCapacity = 5000 * MathF.Pow((matterBinRating - 1), 2);
+            component.HeatCapacity = 5000 * MathF.Pow(matterBinRating, 2);
 
             switch (component.Mode)
             {
-                // 573.15K with stock parts.
+                // 1172.6K with stock parts.
                 case ThermoMachineMode.Heater:
-                    component.MaxTemperature = Atmospherics.T20C + (component.InitialMaxTemperature * laserRating);
+                    component.MaxTemperature = component.BaseMaxTemperature + component.MaxTemperatureDelta * laserRating;
+                    component.MinTemperature = Atmospherics.T20C;
                     break;
-                // 73.15K with stock parts.
+                // 193.15K / -100C with stock parts.
```

Comment:
this needs to be lower, since stock freezers are intended to be good enough for cryo tubes. its 73.15k in SS13 but can probably be a little higher

Reply:
will lower it